tutorials/01_intro.md

```diff
@@ -2,18 +2,61 @@
 
 Next Tutorial: \ref installation
 
-Ignition Physics is a component in Ignition Robotics, a set of libraries
-designed to rapidly develop robot and simulation applications. The main
-goal of the library is to provide an abstraction layer to various physics
-engines, which gives end users the ability to use multiple physics engines
-with minimal change to their application code. Ignition Physics uses
-a plugin architecture where each physics engine is implemented as a plugin
-that can be loaded at runtime. To enable users the ability to choose
-between physics engines, Ignition Physics introduces [\"Features\"](@ref ignition::physics::Feature), a design concept used to encode the
-capabilities of a physics engine. Users can request for physics engines
-that support a set of features and the plugin loading mechanism loads only
-the engines that implement the requested features.
-
-The following is a list of currently supported physics engines:
-
-  * [DART](http://dartsim.github.io/)
+## Motivation
+
+Ignition Physics (`ign-physics`) is a library component in Ignition, a set of libraries designed to rapidly develop robot and simulation applications.
+The main goal of the library is to provide an abstraction layer to various physics engines, which gives end-users the ability to use multiple physics engines with minimal change to their application code.
+Users can request for physics engines that support a set of features and the plugin loading mechanism loads only the engines that implement the requested features.
+Besides, a user-selected physics engine can be integrated into the existing code-base by writing a compatible plugin interface, which enables Ignition Physics extensibility and modularity.  
+
+## Overview
+
+For a big picture of the Ignition Physics operation in Ignition ecosystem, see the abstract diagram below:
+
+<img src="https://user-images.githubusercontent.com/18066876/94050480-9dc54000-fdd6-11ea-92e7-832dcdf4caba.png"/>
```

Comment:
Thanks for the nice diagram! The part about ign-transport and ign-msgs being used for communication makes it appear that ign-physics communicates, for example, with ign-sensors via ign-transport. However, this is not quite the case. In the current versions of Ingition, only ign-gazebo interfaces with ign-physics via it's C++ API. ign-gazebo received state data from ign-physics and sends them to other libraries either directly or via ign-transport. For the particular libraries in the diagram (ign-sensors and ign-rendering), ign-gazebo sends the information directly via their C++ API.

Reply:
@azeey Thanks for the explanations :) Then I will remove the communication description between `ign-physics` with other libraries.